pom.xml

```diff
@@ -35,10 +35,16 @@
   <description>Hawkular Commons is a collection of common code and frameworks used by the Hawkular project and sub-projects.</description>
   <url>http://www.hawkular.org/</url>
   <inceptionYear>2014</inceptionYear>
+
+  <properties>
+    <version.org.freemarker>2.3.23</version.org.freemarker>
```

Comment:
No, not yet. I think this could fit a future PR for parent and a follow up update to commons.